tests/Generator/Instantiator/Chainable/StaticFactoryInstantiatorTest.php

```diff
@@ -244,7 +244,7 @@ public function testThrowsAnExceptionIfFixtureClassDoesNotMatchObjectClass()
 
     /**
      * @expectedException \Nelmio\Alice\Throwable\Exception\Generator\Instantiator\InstantiationException
-     * @expectedExceptionMessage Instantiated fixture was expected to be an instance of "Nelmio\Alice\Entity\Instantiator\DummyWithFakeNamedConstructor". Got "Nelmio\Alice\Throwable\Exception\Generator\Instantiator\InstantiationExceptionFactory" instead.
+     * @expectedExceptionMessage Instantiated fixture was expected to be an instance of "Nelmio\Alice\Entity\Instantiator\DummyWithFakeNamedConstructor". Got "null" instead.
```

Comment:
which makes perfect sense, I think I didn't read the message properly when I copy/pasted it there :)